cumulusci/tasks/salesforce/namespaces.py

```diff
@@ -0,0 +1,61 @@
+from cumulusci.core.utils import process_bool_arg
+from cumulusci.utils import inject_namespace
+from typing import Optional, Tuple
+
+namespace_injection_options = {
+    "managed": {
+        "description": "If False, changes namespace_inject to replace tokens with a blank string",
+        "required": False,
+    },
+    "namespaced_org": {
+        "description": "If True, the tokens %%%NAMESPACED_ORG%%% and ___NAMESPACED_ORG___ will get replaced with the namespace.  The default is false causing those tokens to get stripped and replaced with an empty string.  Set this if deploying to a namespaced scratch org or packaging org.",
+        "required": False,
+    },
+    "namespace_inject": {
```

Comment:
I copied the `namespace_inject` option from [Deploy](https://github.com/SFDO-Tooling/CumulusCI/blob/main/cumulusci/tasks/salesforce/Deploy.py) for consistency.   IMHO, the task name is weird.  Could we instead just call it `namespace` or `namespace_to_inject`?